test/graph_input_tests.cpp

```diff
@@ -507,6 +507,72 @@ TEST_CASE("read_graph, market_matrix")
     }
 }
 
+TEST_CASE("read", "binary")
+{
```

Comment:
maybe add the name of the file you want to read for better error handling

Reply:
Done.